test/gcs_taskrun_test.go

```diff
@@ -0,0 +1,184 @@
+// +build e2e
+
+/*
+Copyright 2018 Knative Authors LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
+	tb "github.com/knative/build-pipeline/test/builder"
+	knativetest "github.com/knative/pkg/test"
+	"github.com/knative/pkg/test/logging"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TestStorageTaskRun is an integration test that will verify GCS input resource runtime contract
+// - adds volumes with expected name
+// - places files in expected place
+
+func TestStorageTaskRun(t *testing.T) {
+	logger := logging.GetContextLogger(t.Name())
+
+	if _, checkgcloudErr := exec.LookPath("gcloud"); checkgcloudErr != nil {
+		logger.Infof("gcloud is not on path. So skipped test")
+		t.Skip()
```

Comment:
could be good to update https://github.com/knative/build-pipeline/blob/master/test/README.md#setup to explain that some tests will be skipped if `gcloud` isn't present

maybe the docs could explain also why you need to have gcloud locally to run this test (id assume that since gcloud is invoked in the cluster, you wouldnt need it locally? looks like maybe it's being used for some test setup tho?)

Reply:
Yes test uses gcloud to create service account. I did not realize another test did something similar. Probably I can update the test to bank on existing test setup.